var/spack/repos/builtin/packages/py-amqp/package.py

```diff
@@ -11,14 +11,20 @@ class PyAmqp(PythonPackage):
 
     pypi = "amqp/amqp-2.4.1.tar.gz"
 
-    license("BSD-3-Clause")
-
```

Comment:
You are removing the license from all 3 files: Please keep it as well.

Please review your diff / changes before commit with `git diff` and before pushing with `git log -p`.

Reply:
I put back the license to all files.